Remove doc/irstlm-manual.pdf from install in --prefix to instead obey --docdir

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,5 +1,3 @@
-documentationdir = @prefix@/doc
-
 if DOC_COMPILATION
 
 irstlm-manual.pdf: irstlm-manual.tex
@@ -14,10 +12,6 @@ clean-local:
 
 all: irstlm-manual.pdf
  
-dist_documentation_SCRIPTS = irstlm-manual.pdf
+dist_doc_DATA = irstlm-manual.pdf
 
 endif
-
-EXTRA_DIST = irstlm-manual.tex 
-
-
